build.gradle

```diff
@@ -15,7 +15,7 @@ ext {
     kelpieVersion = '1.2.3'
     resilience4jRetryVersion = '1.7.1'
     slf4jVersion = '1.7.25'
-    scalarDbVersion = '3.7.2'
+    scalarDbVersion = '3.10.1'
```

Comment:
I think `scalardb-benchmarks` should also be versioned and make it match the ScalarDB version.
So, for example, `scalardb- benchmarks` v3.10.x works for `scalardb` v3.10.x, and `scalardb- benchmarks` v3.9.x works for `scalardb` v3.9.x.

@brfrn169 What do you think?

Reply:
As discussed, let's use the snapshot version here for the master branch. Also, I will prepare the specific branch for v3.10.

Reply:
@jnmt 
The ScalarDB Cluster Java client SDK is published in the Maven Central and the repositories:

Maven Central:
https://central.sonatype.com/artifact/com.scalar-labs/scalardb-cluster-java-client-sdk

Maven Central Snapshot:
https://oss.sonatype.org/content/repositories/snapshots/com/scalar-labs/scalardb-cluster-java-client-sdk/

Maybe can we use them in this benchmark project?

Reply:
@brfrn169 Thank you for the notification. Honestly, I totally forgot it. I will resume working on it as soon as I get a chance.